Tests/SentryCrashTests/SentryStacktraceBuilderTests.swift

```diff
@@ -14,6 +14,11 @@ class SentryStacktraceBuilderTests: XCTestCase {
 
     private var fixture: Fixture!
 
+    override class func setUp() {
+        super.setUp()
+        SentryLog.configure(true, diagnosticLevel: .debug)
+    }
+
```

Comment:
TODO: i can remove all the logging here now that we figured out what the latent bug was that was causing this test case to fail after i made changes. 

I'm scratching my head trying to figure out how the test hadn't failed before 🤔 I didn't make any changes to any SDK logic as part of this PR. I did fix the bug in this PR though, see https://github.com/getsentry/sentry-cocoa/pull/2899/files#diff-84fe6a903bd6efcc5209502d4c66c80e68a8313d0d94927cb347d3b0015eda1fR49